firebase/src/core/domain/use-cases/collections/sync-collections.ts

```diff
@@ -127,4 +143,59 @@ export class SyncCollectionsUseCase {
 
     return [updateableMemos, deletableMemosIds];
   }
+
+  /**
+   * Returns the changed (added, updated and removed) collections ids.
+   *
+   * The collections diff is made by using `git ...` shell commands, so make sure to run the current use-case in the
+   * expected git HEAD.
+   */
+  async #collectionsDiff(): Promise<UpdatedCollections> {
```

Comment:
```suggestion
  async #diffCollections(): Promise<UpdatedCollections> {
```
following the other private function signature